db/schema.sql

```diff
@@ -1,94 +1,104 @@
 -- We can use surah_number to make an api request to https://api.quran.com/api/v4/chapters/(surah_number)
 -- But because I'd prefer to store everything for ourselves, we should make the request to that api only once
 -- and store them here.
-CREATE TABLE Surah (
-    surah_number INTEGER PRIMARY KEY, -- This is the number corresponding to the surah in question
-    revelation_place VARCHAR(20),
-    revelation_order INTEGER,
+DROP TABLE IF EXISTS `Surah`; 
```

Comment:
Please make sure that the docker script works after changing the db schema. The back ticks aren't understood by psql in this context.
See the error in the following picture.
![image](https://user-images.githubusercontent.com/29666432/178886969-13f2d30d-434f-421f-8650-1e86630bbc7f.png)